README.md

```diff
@@ -17,10 +17,10 @@ To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-Ruby: 2.6.3
-Rails: 5.2.3
-Postgres: >=9.5
-mini_racer #optional
+- Ruby: 2.6.3
+- Rails: 5.2.3
+- Postgres: >=9.5
+- mini_racer #optional
```

Comment:
- Update command for running `rspec`. I got this error: `check_for_activated_spec!: You have already activated mini_mime 1.0.3, but your Gemfile requires mini_mime 1.0.2. Prepending `bundle exec` to your command may solve this`. This was fixed by running `bundle exec rspec`